<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1042</width>
    <height>625</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 2, 10);</string>
   </property>
   <widget class="QLabel" name="MainLabel">
    <property name="geometry">
     <rect>
      <x>55</x>
      <y>10</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(211, 224, 255);</string>
    </property>
    <property name="text">
     <string>Mandelbrot</string>
    </property>
   </widget>
   <widget class="QWidget" name="glow" native="true">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>550</y>
      <width>0</width>
      <height>3</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(108, 173, 255);</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>300</width>
      <height>211</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-width: 5px;
border-radius: 10px;
border: rgb(150,150, 150);
border-width: 1px;
background-color: rgb(15, 15, 25);</string>
    </property>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>261</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>22</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background:rgb(146, 150, 255);
color: rgb(0, 0, 0);
border-color: rgb(0, 0, 0);
border-width: 5px;
border-radius: 10px;
border-color:rgb(146, 150, 255);</string>
     </property>
     <property name="text">
      <string>build</string>
     </property>
    </widget>
    <widget class="QLabel" name="MainLabel_2">
     <property name="geometry">
      <rect>
       <x>19</x>
       <y>15</y>
       <width>261</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(211, 224, 255);</string>
     </property>
     <property name="text">
      <string>Set build</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>80</y>
       <width>201</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-width: 10px;
border-radius: 10px;
border: rgb(50,50, 50);
border-style: solid;
border-width: 2px;
background-color: rgb(0, 0, 0);
color: rgb(211, 224, 255);</string>
     </property>
    </widget>
    <widget class="QLabel" name="MainLabel_3">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>83</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>37</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(211, 224, 255);</string>
     </property>
     <property name="text">
      <string>n</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>300</y>
      <width>300</width>
      <height>230</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-width: 5px;
border-radius: 10px;
border: rgb(150,150, 150);
border-width: 1px;
background-color: rgb(15, 15, 25);</string>
    </property>
    <widget class="QLabel" name="MainLabel_4">
     <property name="geometry">
      <rect>
       <x>19</x>
       <y>15</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(211, 224, 255);</string>
     </property>
     <property name="text">
      <string>Information</string>
     </property>
    </widget>
    <widget class="QLabel" name="MainLabel_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(211, 224, 255);</string>
     </property>
     <property name="text">
      <string>Scale:</string>
     </property>
    </widget>
    <widget class="QLabel" name="MainLabel_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(211, 224, 255);</string>
     </property>
     <property name="text">
      <string>Position:</string>
     </property>
    </widget>
    <widget class="QLabel" name="MainLabel_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(211, 224, 255);</string>
     </property>
     <property name="text">
      <string>Black:</string>
     </property>
    </widget>
    <widget class="QLabel" name="MainLabel_8">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(211, 224, 255);</string>
     </property>
     <property name="text">
      <string>Colored:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>80</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(211, 224, 255);</string>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>110</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(211, 224, 255);</string>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>140</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(211, 224, 255);</string>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>170</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(211, 224, 255);</string>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_3" native="true">
    <property name="geometry">
     <rect>
      <x>355</x>
      <y>70</y>
      <width>600</width>
      <height>460</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-width: 5px;
border-radius: 10px;
border: rgb(150,150, 150);
border-width: 1px;
background-color: rgb(15, 15, 25);</string>
    </property>
    <widget class="QGraphicsView" name="myView">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>560</width>
       <height>420</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-width: 5px;
border-radius: 10px;
</string>
     </property>
    </widget>
    <widget class="QWidget" name="kostil" native="true">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>560</width>
       <height>420</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 73, 89);</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="choose" native="true">
    <property name="geometry">
     <rect>
      <x>-800</x>
      <y>-800</y>
      <width>120</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>570</y>
      <width>121</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>34</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(211, 224, 255);</string>
    </property>
    <property name="text">
     <string>100%</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
